### ISSUE TITLE
Source Map wrong path

### ISSUE THREAD

**Issue by PakitoSec** [crcn/sift.js#242]
Hello,

Is it possible to add source map at the right place ?
warning message with react-script V5.0.0

```
WARNING in ./node_modules/sift/es5m/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '.../client/node_modules/sift/src/core.ts' file: Error: ENOENT: no such file or directory, open '..../client/node_modules/sift/src/core.ts'
```

Thanks for your help,

**Comment by crcn** [comment]
I see. Looks like I just need to include the actual source code with the node module. Will push this change here in a sec.

**Comment by crcn** [comment]
Should be fixed in v16.0.0. Let me know if you're still running into this issue!
